spec/services/directions_service.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe DirectionsService do
+  it 'will get directions to the station', :vcr do
+    address = "1331 17th St LL100, Denver, CO 80202"
+    lat = 39.750284
+    long = -104.995227
+    directions = DirectionsService.get_directions(address, lat, long)
+
+    expect(directions).to be_a Hash
+  end
+end
```

Comment:
you can also test the values that come back with this request. Make sure that any of the keys that your code relies on are in the place that your code expects them to be.

Reply:
Similar to what you did in the Station Service spec